src/unix/linux_like/linux/gnu/b64/powerpc64/mod.rs

```diff
@@ -190,6 +194,47 @@ s! {
         pub ss_flags: ::c_int,
         pub ss_size: ::size_t
     }
+
+    #[repr(C)]
+    pub struct vscr_t {
+        #[cfg(target_endian = "big")]
+        __pad: [::c_uint; 3],
+        #[cfg(target_endian = "big")]
+        pub vscr_word: ::c_uint,
+
+        #[cfg(target_endian = "little")]
+        pub vscr_word: ::c_uint,
+        #[cfg(target_endian = "little")]
+        __pad: [::c_uint; 3],
+    }
```

Comment:
glibc source: https://github.com/bminor/glibc/blob/master/sysdeps/unix/sysv/linux/powerpc/sys/ucontext.h#L96-L108

```c
/* Container for Altivec/VMX Vector Status and Control Register.  Only 32-bits
   but can only be copied to/from a 128-bit vector register.  So we allocated
   a whole quadword speedup save/restore.  */
typedef struct _libc_vscr
{
#if __BYTE_ORDER__ == __ORDER_BIG_ENDIAN__
	unsigned int __pad[3];
	unsigned int __ctx(vscr_word);
#else
	unsigned int __ctx(vscr_word);
	unsigned int __pad[3];
#endif
} vscr_t;
```

Reply:
My code is a bit ugly because I have to declare the `target_endian` condition for every single line. I could see there is `align_const!` macro which does it a bit more elegant but I don't see any way that I could use `align_const!` macro inside the `s!` macro. Any suggestion is greatly appreciated.